### PR TITLE
Feat/add technical collective #248

### LINKED ISSUE #248: [Governance] Create build with minimum democracy features required
Starting with pallet_democracy, 
Make a minimalist branch with governance enabled that will build and run.

## Acceptance criteria

- [x] pallet_democracy is added to frequency plus any pallets it depends on
- [x] pallet_collective is added
- [x] weights generated for each new frame pallet
- [x] WeightInfo that uses the new weights is defined in the frame pallet config
- [x] define the Frequency Council with one member to fulfill requirements of pallet_democracy
- [ ] define the Technical Council with one member

### PR DESCRIPTION
**Pull request by shannonwells**
# Goal
The goal of this PR is to add a Technical Committee collective and to set up Council and Technical Committee to be used in the different chain specs.

Closes #248

# Discussion
This is the last item of #248.

# Checklist
- [x] Chain spec updated
- [ ] Updated the js/api-augment code if a custom RPC added/changed
- [x] Design doc(s) updated N/A
- [x] Tests added  N/A
- [x] Benchmarks added N/A
- [x] Weights NOT updated - collective benchmarks panic; need to investigate

### CODE REVIEW

**Review by wilwade — CHANGES_REQUESTED** [review]
One question and one extra file, but no issues elsewhere. Thanks for doing this!

File: runtime/frequency-rococo/src/lib.rs [hunk 1]
```diff
@@ -718,14 +734,17 @@ construct_runtime!(
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T> }= 4,
 		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 5,
 		Democracy: pallet_democracy::{Pallet, Call, Config<T>, Storage, Event<T> } = 6,
-		Council: pallet_collective::<Instance1>::{Pallet, Call, Config<T,I>, Storage, Event<T>, Origin<T>} = 7,
```
[the remark this answers is not in the document]
Comment (shannonwells) [reply runtime/frequency-rococo/src/lib.rs:722]: It's not, it's just an enum value.  There are other numbers being skipped if you look further down, and the indexing is a way to ensure that things get loaded in a [specific order] (https://paritytech.github.io/substrate/master/frame_support/macro.construct_runtime.html), generally of increasing dependency:
> = $n optional: number to define at which index the pallet variants in OriginCaller, Call and Event are encoded, and to define the ModuleToIndex value.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
[the remark this answers is not in the document]
Comment (shannonwells) [reply package-lock.json:1]: whoops! good catch, thanks

**Review by enddynayn — APPROVED** [review]

**Comment by enddynayn** [comment]
types?

**Pull request merged**